test/tests/functional/pbs_preemption.py

```diff
@@ -263,12 +267,16 @@ def test_vnode_resource_contention(self):
                                   mom=self.mom, usenatvnode=False)
 
         a = {'Resource_List.select': '1:ncpus=2+1:ncpus=2'}
-        j = Job(TEST_USER, attrs=a)
-        jid1 = self.server.submit(j)
-        jid2 = self.server.submit(j)
-        jid3 = self.server.submit(j)
-        jid4 = self.server.submit(j)
-        jid5 = self.server.submit(j)
+        j1 = Job(TEST_USER, attrs=a)
+        jid1 = self.server.submit(j1)
+        j2 = Job(TEST_USER, attrs=a)
+        jid2 = self.server.submit(j2)
+        j3 = Job(TEST_USER, attrs=a)
+        jid3 = self.server.submit(j3)
+        j4 = Job(TEST_USER, attrs=a)
+        jid4 = self.server.submit(j4)
+        j5 = Job(TEST_USER, attrs=a)
+        jid5 = self.server.submit(j5)
```

Comment:
Put this into a loop.  Create the job, submit the job, and then check if it is in the R state.  You don't use the job ids after checking if they are in the R state.